tests/tests/cpld_bmark_mpi_p8

```diff
@@ -1,142 +0,0 @@
-#
-#  cpld_bmark_p8 test WW3 mpi
```

Comment:
Why is this test deleted?

Reply:
It was never intended to be a long term test and it was one less test to add aerosols to.  I can add it back if it's desired.